.github/ISSUE_TEMPLATE/infrastructure.md

```diff
@@ -0,0 +1,22 @@
+---
+name: I want to file a bug or feature request to the infrastructure
+about: As a Flutter contributor, you have run into problems with the build/test/release
+  infrastructure, including the build and performance dashboards, devicelab, LUCI etc. Or you have
+  new ideas around those systems.
+title: ''
+labels: 'team: infra'
+assignees: ''
+
+---
+
+<!--  Thank you for contributing to Flutter!
+
+      If you are filing a bug, please add the steps to reproduce, expected and actual results.
+
+      If you are filing a feature request, please describe the use case and a proposal.
+
+      If you are requesting a small infra task with P0 or P1 priority, please add it to the
+      "Infra Ticket Queue" project with "New" column, explain why the task is needed and what
```

Comment:
This could be automated if we had a way to filter issues filed from this template. We could create a label like "infra: ticket" or add [Infra Ticket] to the issue title.

Reply:
This template covers every new infra issue, not only the ticket queue. And adding "Infra Ticket Queue" project doesn't take more effort than adding a label or customized title.

Reply:
Ahh I see. I was under the impression this template was being used only for infra tickets. I would clarify what the difference is between infra ticket issues and normal infra issues.